cmd/ocm-status-addon/main.go

```diff
@@ -30,19 +33,31 @@ func main() {
 	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 
-	logs.AddFlags(logs.NewLoggingConfiguration(), pflag.CommandLine)
+	features := featuregate.NewFeatureGate()
+	err := features.Add(map[featuregate.Feature]featuregate.FeatureSpec{
+		logs.ContextualLogging: {
+			Default:    true,
+			PreRelease: featuregate.Alpha}})
+	if err != nil {
+		panic(err)
+	}
+	logConfig := logs.NewLoggingConfiguration()
+	logs.AddFlags(logConfig, pflag.CommandLine)
 
```

Comment:
I have the same question I posed in #59, why the need for a feature gate for logging? Is this a requirement to use kube contextual logging ?

Reply:
Let's discuss #59 in #59.